third-party/pjproject/patches/config_site.h

```diff
@@ -86,7 +86,7 @@
 /* Increase limits to allow more formats */
 #define	PJMEDIA_MAX_SDP_FMT   64
```

Comment:
The SDP in your log has 50 formats which is getting close to t4he max of 64.  How about increasing PJMEDIA_MAX_SDP_FMT from 64 to 72 while you're at it.  This way we'll be safe for a while.

Reply:
I have increased the number of allowed formats to 72 and updated the commit / description.